scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
```

Comment:
Yeah. Probably functions and comments that are implementation instructions for mixins as well. Will change on another PR.